src/main/resources/templates/ward.html

```diff
@@ -29,6 +29,54 @@ <h4>Ward</h4>
     <b>Code:</b> <span th:text="${ward.fullCode}"></span><br />
     <b>Name:</b> <span th:text="${ward.name}"></span><br />
   </div>
+
+  <h2>Polling Units</h2>
+  <div>
+    <table class="striped responsive-table">
+      <tr>
+        <th>#</th>
+        <th><a th:href="@{?sortBy=fullCode&sortDirection=ASC}">Code</a></th>
+        <th><a th:href="@{?sortBy=description&sortDirection=ASC}">Description</a></th>
+        <th><a th:href="@{?sortBy=longitude&sortDirection=ASC}">Longitude</a></th>
+        <th><a th:href="@{?sortBy=latitude&sortDirection=ASC}">Latitude</a></th>
+      </tr>
+      <tr th:each="pollingUnit, i: ${form.wardPage.content}">
+        <td th:text="${(form.wardPage.number*form.wardPage.size)+i.index+1}"></td>
+        <td style="white-space: nowrap;">
+          <a href="pollingunit.html" th:href="@{/pollingunit/{pollingunitId}(pollingunitId=${pollingUnit.id})}">
+            <span th:text="${pollingUnit.fullCode}"></span>
+          </a>
+        </td>
+        <td th:text="${pollingUnit.description}"></td>
+        <td style="white-space: nowrap;" th:text="${pollingUnit.longitude}"></td>
```

Comment:
Same issue as last

Reply:
Fixed in commit [a1e7951](https://github.com/bioye/naijagov/commit/a1e795154d7813c8448dfd349695fe3b0ddcf088)